log/logger.go

```diff
@@ -0,0 +1,53 @@
+package log
+
+import (
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+
+	pkgerrors "github.com/pkg/errors"
+)
+
+// File returns a logger that writes to a file.
+func File(path string) (logger *log.Logger, loggerClose func(), err error) {
+	if err := os.MkdirAll(filepath.Dir(path), 0755); err != nil {
+		return nil, nil, pkgerrors.WithStack(err)
+	}
+
+	file, err := os.OpenFile(path, os.O_APPEND|os.O_RDWR|os.O_CREATE, 0644)
+	if err != nil {
+		return nil, nil, pkgerrors.WithStack(err)
+	}
+	loggerClose = func() {
+		if err := file.Close(); err != nil {
+			panic(err)
```

Comment:
Sure its problematic if the log file cannot be closed, but we should maybe not panic. A log file is not critical.

Same for other panics in this file.

Reply:
When fie closing throws an error, there is something seriously wrong with the system. Don't think we should complicate the API unnessary.